Automattic/jetpack
# Contact Form: add filter to allow customizing email headers.

See #631

#### Changes proposed in this Pull Request:

Allowing one to customize the email headers used for the emails sent out for each contact form would allow cc'ing or bcc'ing specific addresses, for example.

#### Jetpack product discussion

* No

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Add a new form to one of your posts.
* Add the following snippet to your site using a functionality plugin:
```php
add_filter(
	'jetpack_contact_form_email_headers',
	function ( $headers, $comment_author, $reply_to_addr, $to ) {
		$headers .= 'Bcc: ' . $reply_to_addr . "\r\n";
		return $headers;
	},
	10,
	4
);
```
* Submit a form using the form you've just published.
* An email should be sent to the email address you specified in the address you specified in the snippet above.
